### ISSUE TITLE
exec: "myPythonScript.py": executable file not found in $PATH

### ISSUE THREAD

**Issue by mickours** [djosephsen/skinnywhale#3]
When I tried to run the example from the readme I get this error:

```
docker run 1273432d34b0
exec: "myPythonScript.py": executable file not found in $PATH
Error response from daemon: Cannot start container d3c1c3b9d9cb231dcadc93ea75b7c2dfff935e71f15576d38b7a4154a466106d: [8] System error: exec: "myPythonScript.py": executable file not found in $PATH
```

My script looks like this:

```
$ cat myPythonScript.py 
> #!/usr/bin/python
> print ("Hello World!")
```

I don't know if it is because there is no bash on the "skinnifyied" image but the shebang do no work.

However when I changed the Dockerfile CMD from `CMD ["myPythonScript.py"]` to `CMD ["/usr/bin/python", "myPythonScript.py"]` it works!

I think adding this has a limitation and update the readme would be great.

Also  I'm interested on why the shebang is not working. Any idea?

**Comment by djosephsen** [comment]
ah, yes. thanks for the update on the CMD syntax. I gave up a while ago trying to make various combinations of CMD and ENTRYPOINT work for the Java runtime images I'm using at work.  I find it more expedient to just specify the command in the docker run invocation instead like `docker run java -jar myjar.jar`.  I'll update the doc to match your cmd invocation. 

It's certainly true that /bin/sh (bash, zsh or otherwise) wont exist in the skinnified image, and therefore there isn't anything to interpret the shebang line at the top of the script, so either way (whether you're using CMD in a dockerfile, or you're specifying manually via `docker run`),  you'll have to invoke the interpreter directly.

I'll also note that the NOCLEANUP option exists so you can manually copy things like bash across from the base image to the skinnyified image if you want to.

**Comment by djosephsen** [comment]
readme fixed by 83d9b1060f715f0aa84726cf1d574358138dcf1b
